snuba/admin/jwt.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Optional
+
+import structlog
+
+from snuba.admin.user import AdminUser
+
+logger = structlog.get_logger().bind(module=__name__)
+
+
+CERTS: Optional[Any] = None
+AUDIENCE: Optional[str] = None
+
+
+def _certs() -> Any:
+    """Returns a dictionary of current Google public key certificates for
+    validating Google-signed JWTs. Since these change rarely, the result
+    is cached on first request for faster subsequent responses.
+    """
+    import requests
+
+    global CERTS
+    if CERTS is None:
+        response = requests.get("https://www.gstatic.com/iap/verify/public_key")
+        CERTS = response.json()
+    return CERTS
+
+
+def _get_metadata(item_name: str) -> str:
+    """Returns a string with the project metadata value for the item_name.
+    See https://cloud.google.com/compute/docs/storing-retrieving-metadata for
+    possible item_name values.
+    """
+    import requests
+
+    endpoint = "http://metadata.google.internal"
+    path = "/computeMetadata/v1/project/"
+    path += item_name
```

Comment:
I think i might still be confused by `item_name` is that like the project `attributes` https://cloud.google.com/compute/docs/metadata/default-metadata-values#project-attributes-metadata ?

[edit]: I see it's project id or the numeric project id